docs/conceptual/whats-new-in-visual-fsharp.md

```diff
@@ -39,7 +39,7 @@ Learn about what's new in Visual F#. On this page we only cover the most recent
 
 #### Language, compiler, runtime, interactive
 
-* Normalization and expansion of `Array`, `List`, and `Seq` modules
+* Normalization and expansion of `Array`(Collections.Array-Module-%5BFSharp%5D.md), `List`(Collections.List-Module-%5BFSharp%5D.md), and `Seq`(Collections.Seq-Module-%5BFSharp%5D.md) modules
```

Comment:
I believe you also want to wrap the actual strings (such as `Array`) in square brackets to make sure that those work as links.